src/Storefront/Resources/views/storefront/component/pseudo-modal.html.twig

```diff
@@ -1,26 +1,40 @@
-<div class="js-pseudo-modal-template">
-    <div class="modal fade"
-         tabindex="-1"
-         role="dialog">
-        <div class="modal-dialog"
-             role="document">
-            <div class="modal-content">
-                <div class="modal-header only-close">
-                    <h5 class="modal-title js-pseudo-modal-template-title-element"></h5>
-                    <button type="button"
-                            class="modal-close close"
-                            data-dismiss="modal"
-                            aria-label="Close">
-                        {% block product_detail_zoom_modal_close_button_content %}
```

Comment:
you cant just rename the block as that may cause plugins to break. It would probably be the easiest to just leave the name as it is.